examples/features/health/client/main.go

```diff
@@ -4,65 +4,98 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"io"
 	"log"
 	"time"
 
 	"google.golang.org/grpc"
-	healthpb "google.golang.org/grpc/health/grpc_health_v1"
+	pb "google.golang.org/grpc/examples/features/proto/echo"
+	_ "google.golang.org/grpc/health"
+	"google.golang.org/grpc/resolver"
 )
 
-var (
-	addr   = flag.String("addr", "localhost:50051", "the address to connect to")
-	system = "" // empty string represents the health of the system
+var serviceConfig = `{
+	"loadBalancingPolicy": "round_robin",
+	"healthCheckConfig": {
+		"serviceName": ""
+	}
+}`
+
+const (
+	exampleScheme      = "example"
+	exampleServiceName = "health.example.grpc.io"
 )
 
-func callCheck(client healthpb.HealthClient) {
-	response, err := client.Check(context.Background(), &healthpb.HealthCheckRequest{
-		Service: system,
-	})
+var addrs = []string{"localhost:50051", "localhost:50052"}
 
-	if err != nil {
-		log.Fatalf("client.Check(_) = _, %v: ", err)
+func callUnaryEcho(c pb.EchoClient) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	r, err := c.UnaryEcho(ctx, &pb.EchoRequest{})
+	if err == nil {
+		fmt.Println("UnaryEcho: ", r.GetMessage())
 	}
```

Comment:
It would be nice to print something in case there is an error, for debugging purposes.  How about unconditionally printing both `r.GetMessage()` and `err`?

And, actually, if you use different periods for the two servers (5s and 10s as in the example), then we do expect to see timeouts when they are both in the unhealthy phase for more than 1s.